fastchat/serve/monitor/topic_clustering.py

```diff
@@ -46,6 +47,8 @@ def read_texts(input_file, min_length, max_length, english_only):
             line_texts = [
                 x["content"] for x in l["conversation"] if x["role"] == "user"
             ]
+        elif "turns" in l:
+            line_texts = l["turns"]
```

Comment:
sorry could you explain this a bit?

Reply:
Yes, input json file has different format. For examples, the 54K prompt json file you sent me uses "turns" to store the conversation prompts. I added support for json files that uses "turns".